root/static/scripts/edit/URLCleanup.js

```diff
@@ -1470,7 +1475,7 @@ const CLEANUPS = {
       new RegExp('^(https?://)?(www\\.)?metal-archives\\.com/(bands?|albums|artists|labels)', 'i'),
       new RegExp('^(https?://)?(www\\.)?spirit-of-metal\\.com/', 'i'),
       new RegExp('^(https?://)?(www\\.)?ibdb\\.com/', 'i'),
-      new RegExp('^(https?://)?(www\\.)?lortel.\\org/', 'i'),
+      new RegExp('^(https?://)?(www\\.)?lortel\\.org/', 'i'),
```

Comment:
This is the only thing here that should be a bug fix rather than a readability/speed improvement, AFAICT